ament_index_cpp/src/has_resource.cpp

```diff
@@ -17,6 +17,7 @@
 #include <fstream>
 #include <stdexcept>
 #include <string>
+#include <rcpputils/filesystem_helper.hpp>
```

Comment:
There should definitely be a blank line between the system headers block and other library headers.